lib/generators/geoblacklight/install_generator.rb

```diff
@@ -131,6 +124,14 @@ def copy_config_vite_json
       run "yarn install"
     end
 
+    def add_frontend
+      if options[:test]
+        run "yarn add file:#{Geoblacklight::Engine.root}"
+      else
+        run "yarn add @geoblacklight/frontend^5.0.0-alpha"
```

Comment:
The `install_generator` only runs once when you create your app, so after this line adds it to your `package.json`, you are free to update it as you like. It will be listed there just like the rest of your dependencies.

Reply:
Oh, I misunderstood: you mean from the geoblacklight maintainer perspective, not from the built app! Yes, we should document this...maybe there is a good place for a "how to update the frontend assets" guide at docs.geoblacklight.org

Reply:
Now that I think about it, the version specifier may not be necessary...or it might only be necessary if the package is an alpha? I think we just always want the latest one

Reply:
Alternatively, if the frontend package version should always match the GBL version, we could use that to figure out which package to install. If you're on `main` it would be the latest

Reply:
Yes! Concern is for maintainers. I feel like this line could easily be overlooked hidden in this file, when other dependencies are all updated via package.json.
Maybe a comment there about removing the alpha specifier after the release?
And just don't want to lose sight of the need for docs. Maybe we can open a ticket for it (good community sprint work?).